api-metastore/src/test/java/org/zalando/nakadi/service/EventTypeServiceTest.java

```diff
@@ -247,20 +243,6 @@ public void testFeatureToggleDisableLogCompaction() {
         eventTypeService.create(eventType, true);
     }
 
-    @Test
```

Comment:
Moved tests from this class to the new class responsible for this behaviour now `SchemaService`.